python/tvm/relay/build_module.py

```diff
@@ -631,7 +652,7 @@ def _make_executor(self, expr=None):
         # generated code.
         temp_so_dir = contrib_utils.TempDirectory()
         temp_so = temp_so_dir / "temp.so"
-        mod.export_library(temp_so, cc="gcc", options=["-std=c11"])
+        mod.export_library(temp_so, cc="c++", options=["-std=gnu++14"])
```

Comment:
how come it's necessary to invoke the c++ compiler?

Reply:
Looks as though its currently -std=c++11, should this be -std=c11?

Reply:
yep. amended